tonic/src/transport/server/mod.rs

```diff
@@ -88,7 +91,29 @@ pub struct Server<L = Identity> {
     http2_keepalive_timeout: Option<Duration>,
     max_frame_size: Option<u32>,
     accept_http1: bool,
-    layer: L,
+    service_builder: ServiceBuilder<L>,
+}
+
+impl Default for Server<Identity> {
```

Comment:
I feel like we shouldn't have to implement this manually, is there something missing in tower?

Reply:
Ah okay huh I guess derive doesn't take into account default bounds